package.json

```diff
@@ -6,14 +6,20 @@
   "license": "Apache 2",
   "repository": "https://github.com/coinalpha/gateway-api",
   "scripts": {
-    "start": "pm2 start babel-node --no-daemon -- src/index.js",
+    "prebuild": "rimraf dist && mkdir dist",
+    "build": "tsc --project ./",
+    "format": "prettier . --write",
+    "lint": "eslint src test --format table --fix",
+    "debug": "DEBUG=*router nodemon --ext 'ts,js,json' --exec 'ts-node' src/index.ts",
+    "start": "pm2 start babel-node --no-daemon -- dist/index.js",
```

Comment:
Babel should be not use for production.

https://babeljs.io/docs/en/babel-node
> Not meant for production use
>
>You should not be using babel-node in production. It is unnecessarily heavy, with high memory usage due to the cache being stored in memory. You will also always experience a startup performance penalty as the entire app needs to be compiled on the fly.
>
>Check out the example Node.js server with Babel for an idea of how to use Babel in a production deployment.

You can use just regular `node`.

When you compile TypeScript code to js, in result you will get source code what you can use with regular `node`.